packages/interledger/src/admin-api/schema.graphql

```diff
@@ -1,22 +1,437 @@
+    first: Int
+    "Paginating backwards: The last **n** elements from the page."
+    last: Int
+  ): IlpAccountsConnection!
+  liquidityAccountId: ID
+  maxPacketAmount: String!
+  http: Http!
+  asset: Asset!
+  stream: Stream!
+  routing: Routing!
+
+  balance: Balance!
+  webhooks(
+    "Paginating forwards: the cursor before the the requested page."
+    after: String
+    "Paginating backwards: the cursor after the the requested page."
+    before: String
+    "Paginating forwards: The first **n** elements from the page."
+    first: Int
+    "Paginating backwards: The last **n** elements from the page."
+    last: Int
+  ): WebhooksConnection!
+  deposits(
+    "Paginating forwards: the cursor before the the requested page."
+    after: String
+    "Paginating backwards: the cursor after the the requested page."
+    before: String
+    "Paginating forwards: The first **n** elements from the page."
+    first: Int
+    "Paginating backwards: The last **n** elements from the page."
+    last: Int
+  ): DepositsConnection!
+  withdrawals(
+    "Paginating forwards: the cursor before the the requested page."
+    after: String
+    "Paginating backwards: the cursor after the the requested page."
+    before: String
+    "Paginating forwards: The first **n** elements from the page."
+    first: Int
+    "Paginating backwards: The last **n** elements from the page."
+    last: Int
+  ): WithdrawalsConnection!
+}
+
+type Http {
+  incoming: HttpIncoming!
+  outgoing: HttpOutgoing!
+}
+
+type HttpIncoming {
+  authTokens: [String!]!
+}
+type HttpOutgoing {
+  authTokens: String!
+  endpoint: String!
+}
+
+type Stream {
+  enabled: Boolean!
+}
+
+type Routing {
+  staticIlpAddress: String!
+  inheritFromRemote: Boolean!
+  dynamicIlpAddress: String
+}
+
+type Balance {
+  id: ID!
+  createdTime: String!
+  asset: Asset!
+  balance: Int!
+  netLiability: Int
+  netAssets: Int
+  creditExtended: Int!
+  totalLent: Int!
+  operator: Operator
+}
+
+type Operator {
+  trustlineId: ID!
+  availableCredit: Int!
+  totalBorrowed: Int!
+}
+
+type WebhooksConnection {
+  pageInfo: PageInfo!
+  edges: [WebhookEdge!]!
+}
+
+type WebhookEdge {
+  node: Webhook!
+  cursor: String!
+}
+
+type Webhook {
+  id: ID!
+}
+
+type DepositsConnection {
+  pageInfo: PageInfo!
+  edges: [DepositEdge!]!
+}
+
+type DepositEdge {
+  node: Deposit!
+  cursor: String!
+}
+
+type Deposit {
+  id: ID!
+  amount: Int!
+  createdTime: Int!
+}
+
+type WithdrawalsConnection {
+  pageInfo: PageInfo!
+  edges: [WithdrawalEdge!]!
+}
+
+type WithdrawalEdge {
+  node: Withdrawal!
+  cursor: String!
+}
+
+type Withdrawal {
+  id: ID!
+  amount: Int!
+  createdTime: Int!
+  finalizedTime: Int
+}
+
+type Trustline {
+  id: ID!
+  createdTime: Int!
+  creditorAccountId: ID
+  debtorAccountId: ID!
+  availableCredit: Int!
+  debtBalance: Int!
+}
+
+type CreateIlpAccountMutationResponse implements MutationResponse {
+  code: String!
```

Comment:
What does `code` refer to in all of these responses? Is it the string error enum?

Reply:
It's meant to represents the status of the data transfer, like an HTTP status code.